src/render/TextRenderer.js

```diff
@@ -41,16 +41,16 @@ define([
          * @alias TextRenderer
          * @constructor
          * @classdesc Provides methods useful for displaying text. An instance of this class is attached to the
-         * WorldWindow {@link DrawContext} and is not intended to be used independently of that. Applications typically do
-         * not create instances of this class.
+         * WorldWindow {@link DrawContext} and is not intended to be used independently of that. Applications typically
+         * do not create instances of this class.
```

Comment:
This apparently superfluous modification was done to amend the commit message (WebStorm won't allow the amend if the code itself is not changed in some way).